src/rust-crypto/rust-crypto.ts

```diff
@@ -202,6 +203,15 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
     //
     ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+    private cryptoMode = CryptoMode.Legacy;
+
+    /**
+     * Implementation of {@link Crypto.CryptoApi#setCryptoMode}.
+     */
+    public setCryptoMode(cryptoMode: CryptoMode): void {
+        this.cryptoMode = cryptoMode;
+    }
```

Comment:
This should be in the `CryptoApi implementation` section, not `CryptoBackend implementation`